packages/nuxt/src/vite/utils.ts

```diff
@@ -26,6 +26,15 @@ export function findDefaultSdkInitFile(type: 'server' | 'client'): string | unde
   return filePaths.find(filename => fs.existsSync(filename));
 }
 
+/**
+ *  Extracts the filename from a path.
+ */
+export function getFilenameFromPath(path: string): string | null {
```

Comment:
l: any reason to use this over `path.basename`?

Reply:
Because it's not only a path, it's the full node command like `node whatever/path/index.mjs`, but I can change the function signature to make it more clear.

Reply:
afaict, `path.basename` handles such a scenario:

![image](https://github.com/user-attachments/assets/5d9333bc-a080-46a9-88ec-3c7d8ff0de22)

but no strong feelings. If we stick with this implementation, let's just make sure we document why we need it instead of using the builtin :)

Reply:
Yeah, it generally works but my tests where I checked for a Windows-style string and for a string like this `node ./server/`, `path.basename` did not work.